src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderDropdown.scss

```diff
@@ -16,6 +16,29 @@
   z-index: 0;
 }
 
+.global-header--list {
```

Comment:
These are fixes with many thanks to @ChitlangeSahas to sleekify the change-default dropdowns, and make them consistent with the figma.